File: chrome/browser/resources/print_preview/print_preview.js
// Copyright 2019 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

import './ui/app.js';

export {createDestinationKey, ColorMode, Destination, DestinationCertificateStatus, DestinationConnectionStatus, DestinationOrigin, DestinationType, makeRecentDestination} from './data/destination.js';
// <if expr="chromeos">
export {BackgroundGraphicsModeRestriction, ColorModeRestriction, DuplexModeRestriction, PinModeRestriction} from './data/destination_policies.js';
// </if>
export {DestinationErrorType, DestinationStore} from './data/destination_store.js';
export {CloudPrintInterface, CloudPrintInterfaceEventType} from './cloud_print_interface.js';
export {InvitationStore} from './data/invitation_store.js';
export {PrinterType} from './data/destination_match.js';
export {NativeLayer} from './native_layer.js';
export {Error, State} from './data/state.js';
export {PreviewAreaState} from './ui/preview_area.js';
export {MeasurementSystem, MeasurementSystemUnitType} from './data/measurement_system.js';
export {setCloudPrintInterfaceForTesting} from './cloud_print_interface_manager.js';
export {PluginProxy} from './ui/plugin_proxy.js';
export {DuplexMode, DuplexType, getInstance, whenReady} from './data/model.js';
export {DestinationState} from './ui/destination_settings.js';
export {SelectBehavior} from './ui/select_behavior.js';
export {CustomMarginsOrientation, Margins, MarginsType} from './data/margins.js';
export {ScalingType} from './data/scaling.js';
export {Size} from './data/size.js';
export {getSelectDropdownBackground} from './print_preview_utils.js';

File: chrome/test/data/webui/print_preview.cc
// Copyright (c) 2012 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

#include "chrome/test/data/webui/print_preview.h"

#include "base/command_line.h"
#include "chrome/common/chrome_switches.h"

PrintPreviewWebUITest::PrintPreviewWebUITest() {}

PrintPreviewWebUITest::~PrintPreviewWebUITest() {}

// Always print preview for testing.
void PrintPreviewWebUITest::SetUpCommandLine(CommandLine* command_line) {
  WebUIBrowserTest::SetUpCommandLine(command_line);
#if !defined(GOOGLE_CHROME_BUILD)
  command_line->AppendSwitch(switches::kEnablePrintPreview);
#endif
}
